.gitignore

```diff
@@ -214,9 +214,9 @@ gfpgan/
 configs/models.yaml
 
 # weights (will be created by installer)
-models/ldm/stable-diffusion-v1/*.ckpt
-models/clipseg
-models/gfpgan
+# models/ldm/stable-diffusion-v1/*.ckpt
```

Comment:
Should these just be removed instead of commented out?

Reply:
Good point.